browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1639,15 +1661,15 @@ Are you sure you want to do this?
       <message name="IDS_VALUE_5_BAT" desc="value for 5 BAT">
         5 BAT
       </message>
-      <message name="IDS_VALUE_10_BAT" desc="value for 10 BAT">
+      <!-- <message name="IDS_VALUE_10_BAT" desc="value for 10 BAT">
         10 BAT
       </message>
       <message name="IDS_VALUE_0_BAT" desc="value for 0 BAT">
         0.00 BAT
       </message>
       <message name="IDS_VALUE_0_USD" desc="value for 0 USD">
         0.00 USD
-      </message>
+      </message> -->
```

Comment:
let's remove all commented strings that we don't need anymore.

Reply:
done